src/main/java/net/mcreator/ui/laf/renderer/elementlist/DetailsIconModListRender.java

```diff
@@ -1,145 +1,143 @@
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.laf.renderer.elementlist;
+
+import net.mcreator.minecraft.MCItem;
+import net.mcreator.ui.component.util.PanelUtils;
+import net.mcreator.ui.init.L10N;
+import net.mcreator.ui.init.TiledImageCache;
+import net.mcreator.ui.init.UIRES;
+import net.mcreator.util.StringUtils;
+import net.mcreator.util.image.ImageUtils;
+import net.mcreator.workspace.elements.IElement;
+import net.mcreator.workspace.elements.ModElement;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class DetailsIconModListRender extends JPanel implements ListCellRenderer<IElement> {
+
+	public DetailsIconModListRender() {
+		setLayout(new BorderLayout(15, 0));
+	}
+
+	@Override
+	public Component getListCellRendererComponent(JList<? extends IElement> list, IElement element, int index,
+			boolean isSelected, boolean cellHasFocus) {
+		removeAll();
+		setBorder(null);
+
+		JLabel label = new JLabel();
+		JLabel label2 = new JLabel();
+		JLabel label3 = new JLabel();
+		JLabel label4 = new JLabel();
+		JLabel label5 = new JLabel();
+
+		JLabel icon = new JLabel();
+		if (element != null) {
+			if (isSelected) {
+				label.setForeground((Color) UIManager.get("MCreatorLAF.DARK_ACCENT"));
+				label.setBackground((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"));
+				label2.setForeground((Color) UIManager.get("MCreatorLAF.DARK_ACCENT"));
+				label2.setBackground((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"));
+				label3.setForeground((Color) UIManager.get("MCreatorLAF.DARK_ACCENT"));
+				label3.setBackground((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"));
+				label4.setForeground((Color) UIManager.get("MCreatorLAF.DARK_ACCENT"));
+				label4.setBackground((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"));
+				label5.setForeground((Color) UIManager.get("MCreatorLAF.DARK_ACCENT"));
+				label5.setBackground((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"));
+				setOpaque(true);
+				setBackground((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"));
+			} else {
+				label.setForeground((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"));
+				label2.setForeground((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"));
+				label3.setForeground((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"));
+				label4.setForeground((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"));
+				label5.setForeground((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"));
+				setOpaque(false);
+			}
+
+			label.setText(StringUtils.abbreviateString(element.getName(), 6));
```

Comment:
Why did you change this part. For details and list, I don't think you need to abbreviate, especially not to 6 characters.

You also removed font setting part from original code. revert this